presidio-anonymizer/README.MD

```diff
@@ -82,7 +80,25 @@ pip install presidio-anonymizer
 
 #### Getting started
 
-[comment]: shhart '#TODO [ADO-2797] Document again as AnonymizerRequest constructor changed'
+```python
+from presidio_anonymizer import AnonymizerEngine
+from presidio_anonymizer.entities import AnalyzerResult, AnonymizerConfig
+
+# Initialize the engine with logger.
+engine = AnonymizerEngine()
+
+# Class the anonymize function with the text, analyzer results and
+# Anonymizers config to define the anonymization type.
+result = engine.anonymize(
+    text="My name is Bond, James Bond",
+    analyzer_results=[AnalyzerResult("PERSON", 11, 15, 0.8),
+                      AnalyzerResult("PERSON", 17, 27, 0.8)],
+    anonymizers_config={"PERSON": AnonymizerConfig("replace", {"new_value": "BIP"})}
+)
+
+print(result)
+
+```
 
 ### As service:
```

Comment:
Not part of this PR, but can you please remove the `pipenv sync` part? Official usage ways are through pip install and Docker, from code is the third option, and it's covered in the installation docs.

Reply:
Done.

Reply:
Could be this doc is missing all the links to those places???

Reply:
Lets talk about this tomorrw :) I think there is more to be done.